CameraBrowser/CameraBrowserSample/CameraBrowserExtension.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using ImageCaptureCore;
+using System.Linq;
+
+namespace CameraBrowserSample
+{
+	public static class CameraBrowserExtension 
+	{
+		public static bool canSyncClock(this ICDevice self) => self.Capabilities.Contains(ICDeviceCapabilities.CameraDeviceCanSyncClock);
+
+		public static bool canTakePicture(this ICDevice self) => self.Capabilities.Contains(ICDeviceCapabilities.CameraDeviceCanTakePicture);
+
+		public static bool canDeleteOneFile(this ICDevice self) => self.Capabilities.Contains(ICDeviceCapabilities.CameraDeviceCanDeleteOneFile);
+
+		public static bool canDeleteAllFiles(this ICDevice self) => self.Capabilities.Contains(ICDeviceCapabilities.CameraDeviceCanDeleteAllFiles);
+
+		public static bool canReceiveFile(this ICDevice self) => self.Capabilities.Contains(ICDeviceCapabilities.CameraDeviceCanReceiveFile);
+
+		public static bool canEject(this ICDevice self) => self.Capabilities.Contains(ICDeviceCapabilities.DeviceCanEjectOrDisconnect);
```

Comment:
PascalCase is also missing in all the following ones.
```suggestion
		public static bool CanTakePicture(this ICDevice self) => self.Capabilities.Contains(ICDeviceCapabilities.CameraDeviceCanTakePicture);

		public static bool CanDeleteOneFile(this ICDevice self) => self.Capabilities.Contains(ICDeviceCapabilities.CameraDeviceCanDeleteOneFile);

		public static bool CanDeleteAllFiles(this ICDevice self) => self.Capabilities.Contains(ICDeviceCapabilities.CameraDeviceCanDeleteAllFiles);

		public static bool CanReceiveFile(this ICDevice self) => self.Capabilities.Contains(ICDeviceCapabilities.CameraDeviceCanReceiveFile);

		public static bool CanEject(this ICDevice self) => self.Capabilities.Contains(ICDeviceCapabilities.DeviceCanEjectOrDisconnect);
```